internal/scheduler/loadbalancealgoimpl.go

```diff
@@ -49,6 +51,14 @@ func (algo *LoadBalanceAlgoImpl) GetNode(nodebindings []*nodebinding.Model, node
 		})
 	}
 
+	rand.Seed(time.Now().UnixNano())
+	rand.Shuffle(
+		len(nodeCountList),
+		func(i, j int) {
+			nodeCountList[i], nodeCountList[j] = nodeCountList[j], nodeCountList[i]
+		},
+	)
+
 	sort.Sort(nodeCountList)
```

Comment:
Wouldn't `sort.Sort` nullify the effects of `rand.Shuffle`?

Reply:
No.. sort will just move the out of order items, but the items having same value will remain as it is